src/Sdk/NewsroomPrivacyRequests/NewsroomPrivacyRequests.ts

```diff
@@ -0,0 +1,40 @@
+import { Newsroom, PrivacyRequest } from '../../types';
+
+import DeferredJobsApiClient from '../DeferredJobsApiClient';
+import routing from '../routing';
+import { NewsroomPrivacyRequestCreateRequest } from './types';
+
+type NewsroomId = Newsroom['uuid'] | Newsroom['id'];
```

Comment:
Since this is a totally new SDK method, shouldn't we force the usage of `uuid` here?
Or are there some scenarios, where only `id` can be provided?

Reply:
No, I just wrote it based on how we use it in other SDKs. But makes sense, we should be getting rid of deprecated `id`. Thanks! 👍🏻